vcell-core/src/main/java/cbit/vcell/model/Kinetics.java

```diff
@@ -1129,14 +1132,14 @@ private ArrayList<KineticsParameter> getKineticsParametersFromTokens(String kine
 		}
 

```

Comment:
All existing formats may not be documented with comments here, we have to tread carefully when we change the parsing logic.

The setName() code seems harmless ... but consider that this functionality used to work - so unless someone previously changed the parsing code - the parsing code is not the problem.